lib/src/arithmetic/values/floating_point_values/floating_point_value_populator.dart

```diff
@@ -70,6 +72,80 @@ class FloatingPointValuePopulator<FpvType extends FloatingPointValue> {
       ..validate();
   }
 
+  /// This is a helper function to canonicalize the sign, exponent, and
+  /// mantissa values for the FloatingPointValue.
+  ({LogicValue sign, LogicValue exponent, LogicValue mantissa}) _components(
```

Comment:
i think this function could be `static`?

Reply:
It needs to know if the current Populator has explicitJBit set.

Reply:
looks like it doesnt?  it only uses `this.exponentWidth` right now?  should it be using `fpv.exponentWidth` instead?